Cargo.toml

```diff
@@ -55,3 +55,6 @@ io-dump = { git = "https://github.com/carllerche/io-dump" }
 rustls = "0.11"
 tokio-rustls = { git = "https://github.com/briansmith/tokio-rustls", tag = "b/p1" }
 webpki-roots = "0.13"
+
+[replace]
+"http:0.1.0" = { git = "https://github.com/hyperium/http" }
```

Comment:
This shouldn't be needed any more, we just published 0.1.1 that includes the changes the tests were relying upon.

Reply:
🙌